.travis.yml

```
@@ -28,11 +34,11 @@ before_script:
   - printenv | sort
   - git show --pretty=fuller -s
   - mkdir -p coverage
-  - etc/travis_install_gap.sh
+  - if [[ -v GAPROOT ]]; then etc/travis_install_gap.sh; fi
```

More idiomatic would be
```suggestion
  - [[ -v GAPROOT ]] && etc/travis_install_gap.sh
```